examples/Example1_Fit_Halo_Mass_to_Shear_Catalog.ipynb

```diff
@@ -176,7 +176,7 @@
    "outputs": [],
    "source": [
     "ideal_data = mock.generate_galaxy_catalog(cluster_m, cluster_z, concentration,\n",
-    "                                          cosmo, ngals, Delta, src_z)"
+    "                                          cosmo, Delta, src_z, ngals=ngals)"
```

Comment:
Relatively minor comment, but it would be good for the markdown at the top of this to describe the difference between this Example, Example2, and Example3.  Example 3 has a nice header.  For Example1, something like, **Fit Halo Mass from a Shear Catalog: The Ideal Data Example**

Reply:
I think I've addressed all of your points above regarding make the differences between Example1-2-3 notebooks easier to grasp. Hope this is suitable.